src/metaschema/oscal_ssp_metaschema.xml

```diff
@@ -1258,13 +1258,13 @@
                </allowed-values>
             </flag>
          </assembly>
-         <assembly ref="set-param" max-occurs="unbounded">
-            <group-as name="set-params" in-json="BY_KEY"/>
+         <assembly ref="set" max-occurs="unbounded">
```

Comment:
It might be more consistent to use `parameter-setting` as the assembly name instead of `set`.